components/core/src/TimestampPattern.cpp

```diff
@@ -235,7 +240,8 @@ bool TimestampPattern::parse_timestamp (const string& line, epochtime_t& timesta
                     }
 
                     int value;
-                    if (!convert_string_to_number(line, line_ix, line_ix + cFieldLength, '0', value) || value < 0 || value > 99) {
+                    if (!convert_string_to_number(line, line_ix, line_ix + cFieldLength, '0',
```

Comment:
Change to `false ==` (or appropriate failure value) to match style guide and be consistent with new changes.

Appears in many other places in this file.
Even though this is existing code, we want to try and fix/improve any file we touch during a commit/PR.

Reply:
commit 425681be9d06531982c55187aeb6cde773e0c058